dev/install_git_hooks.sh

```diff
@@ -0,0 +1,7 @@
+GIT_ROOT=$(git rev-parse --show-toplevel)
+
+cd "$GIT_ROOT" || exit
+
+HOOKS_PATH="$GIT_ROOT/.git/hooks"
+cd $HOOKS_PATH && ln -s ../../dev/githooks/* .
```

Comment:
just a bit neat:
```shell
cd "$HOOKS_PATH" && ln -s "$GIT_ROOT"/dev/githooks/* . 
```
also to prevent glob splitting

Reply:
yeah should use shellcheck to check my script...